src/transformers/models/llava/processing_llava.py

```diff
@@ -30,9 +30,7 @@
 
 class LlavaProcessorKwargs(ProcessingKwargs, total=False):
     _defaults = {
-        "text_kwargs": {
-            "padding": False,
-        },
```

Comment:
This was the default kwarg value initially in the processor, wasn't it? I might be mistaken but I thought it was padding_side that was culprit

Reply:
my bad, saw `padding_side` here somehow hehe, will fix back